src/features/public/Donations/components/TreeDonation.tsx

```diff
@@ -123,7 +130,7 @@ function TreeDonation({
     <>
       <div
         className={styles.cardContainer}
-        style={{ alignSelf: isGift ? 'start' : 'center' }}
+        style={{ alignSelf: isGift ? 'start' : 'center', width: `${screenWidth}px` }}
```

Comment:
Because of the width specified here the design breaks on the screens with size < 412. 

<img width="440" alt="Screenshot 2020-11-19 at 11 02 19 AM" src="https://user-images.githubusercontent.com/25605193/99625658-b78da700-2a56-11eb-8c6d-0442ecfba981.png">

Reply:
If the screen width is below 412, set it to 100% width

Reply:
done.